server/src/internalClusterTest/java/org/opensearch/cluster/coordination/UnsafeBootstrapAndDetachCommandIT.java

```diff
@@ -321,9 +356,10 @@ public void test3MasterNodes2Failed() throws Exception {
         detachCluster(environmentMaster3, false);
 
         logger.info("--> start 2nd and 3rd master-eligible nodes and ensure 4 nodes stable cluster");
-        internalCluster().startMasterOnlyNode(master2DataPathSettings);
```

Comment:
Earlier, it was providing the data path explicitly, why did we remove it?

Reply:
Not able to recall why I removed it but yes I will add it back.